account_followup/__openerp__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2004-2010 Tiny SPRL (<http://tiny.be>)
+# Copyright 2007 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Payment Follow-up Management',
+    'version': '9.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'author': 'Odoo SA,'
+              'Tecnativa',
+    'license': 'AGPL-3',
+    'website': 'https://www.odoo.com/',
+    'depends': [
+        'account',
+        'account_accountant',
```

Comment:
I think we can remove `account_accountant` and keep `account` only.
Module `account_accountant` is only meant to give to Admin user all the accounting rights. We don't need it for this module. I'm wondering why Odoo added this module in the dependencies of `account_followup`.